### DISCUSSION TITLE
Seq.App c# Non-CLEF line received at STDERR: Cannot use file stream ...

### DISCUSSION THREAD

**Discussion by mKamiAsim** [datalust/seq-tickets#1353] [Help]
I wrote Seq app in C# and verified package in development/ staging environment. App is working fine. 
When I deployed same package on production SEQ, and tried to send alert manually, I got following warning message.

Non-CLEF line received from STDERR: Cannot use file stream for [C:\Windows\TEMP\Seq\Client\01xxxxxxxxxxxxxxx\seqcli.depts.json]: No such file or directory

Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value C. Path '', line 0, position 0

This message happens once package is deployed, and then repeats after each alert.
Am I missing some dependencies. as same package is working fine on other environments. 

For reference I am using 
Handlebars.Net 2.0.9
Netwonsoft.Json  12.0.3
Seq.Apps   5.1.0
Serilog 2.10.0
System.Collections.Specialized 4.3.0  (This was added, because seq raised exception assembly not found (only on production))
On production I only installed .Net 5 hosting bundle and Seq 2021.2

Project type .Net standard 2.0

**Comment by nblumhardt** [comment]
Hi Muhummad,

Thanks for getting in touch! It sounds like the contents of Temp has been cleared on the production server; this triggers a known issue: https://github.com/datalust/seq-tickets/issues/1242 which is fixed in the approaching 2021.3 release.

To verify this, and to work around it until the upgrade is available, you can restart the Seq service on the production machine with:

```
seq service restart
```

Let me know if this helps,
Nick
